### PR TITLE
Create Data Confidentiality in Open Science

### PR DESCRIPTION
**Pull request by krig1994**

### CODE REVIEW

File: Data Confidentiality in Open Science [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+industry partners at the project's inception, explicitly defining data ownership, the extent of data sharing, and the conditions under which data can be made publicly 
+accessible. This transparency can help prevent conflicts and ensure compliance with confidentiality requirements.
+2. For sensitive industry data, researchers can employ data anonymization techniques to remove personally identifiable information and confidential details, facilitating
+the sharing of aggregated and de-identified data while preserving data confidentiality.
+3. Researchers, institutions, and industry partners should engage in open dialogue and raise awareness about the benefits of open science. Encouraging industry stakeholders 
+to embrace more open data sharing practices can lead to increased collaboration and innovation.
+4. In certain cases, trusted third-party organizations can act as intermediaries, managing industry data in a manner that ensures compliance with confidentiality
+agreements while enabling as much open data sharing as possible.
+
+IV. COLCLUSION
```
Comment (willu47) [inline Data Confidentiality in Open Science:46]: Typo here - CONCLUSION

File: Data Confidentiality in Open Science [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+category: practice
+tags: reusability
+---
+
+Data Confidentiality in Open Science
+
+I. INTRODUCTION
+
+Open science represents a groundbreaking approach to research, emphasizing transparency, collaboration, and the sharing of knowledge, data, and methodologies within 
+the scientific community. [1] A central pillar of open science is the availability of research data. Nonetheless, ensuring data confidentiality within the framework of
```
Comment (willu47) [inline Data Confidentiality in Open Science:15]: Can you add citations as hyperlinks to the DOI? https://www.markdownguide.org/basic-syntax/#reference-style-links

File: Data Confidentiality in Open Science [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+IV. CONCLUSION
+
+Data confidentiality is a crucial consideration within the realm of open science. While open data sharing is a fundamental principle for advancing scientific knowledge, 
+it is equally important for researchers to respect the confidentiality requirements of industries and data owners. By establishing clear agreements, employing data 
+anonymization techniques, and fostering collaboration and awareness, the scientific community can strive for a harmonious balance that allows the exchange of knowledge
+while safeguarding the interests of all stakeholders involved. Open science, when executed with openness and consideration, can benefit both researchers and industry,
+driving innovation and expanding the frontiers of human knowledge.
+
+
+REFERENCES
```
Comment (willu47) [inline Data Confidentiality in Open Science:55]: For all headings, use a `#` to denote a heading.

File: Data Confidentiality in Open Science [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+1	E. and M. National Academies of Sciences: Open Science by Design: Realizing a Vision for 21st Century Research, 2018, pp.15
```
Comment (willu47) [inline Data Confidentiality in Open Science:57]: Please add DOIs and hyperlinks to each paper (ideally place them inline as a link).  Also, the authors for this first item do not seem correct.

File: Data Confidentiality in Open Science (hunk 1 above)
Comment (krig1994) [reply Data Confidentiality in Open Science]: done

File: Data Confidentiality in Open Science [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  " Data Confidentiality in Open Science"
+date:   2023-11-06
+author: Konstantinos Rigas
+category: practice
+tags: reusability
+---
+
+Data Confidentiality in Open Science
```
Comment (willu47) [inline Data Confidentiality in Open Science:10]: This title is not needed as the title is rendered from the metadata.

**Review by willu47 — CHANGES_REQUESTED** [review]
See inline comments for suggested changes.

File: Data Confidentiality in Open Science (hunk 2 above)
Comment (krig1994) [reply Data Confidentiality in Open Science:15]: hyperlinks to the DOI have been added

File: Data Confidentiality in Open Science (hunk 3 above)
Comment (krig1994) [reply Data Confidentiality in Open Science]: The headings have been denoted

File: Data Confidentiality in Open Science (hunk 4 above)
Comment (krig1994) [reply Data Confidentiality in Open Science]: Fixed

File: Data Confidentiality in Open Science (hunk 5 above)
Comment (krig1994) [reply Data Confidentiality in Open Science]: The title has now been removed

**Review by willu47 — APPROVED** [review]

**Pull request merged**